content/pytorch/concepts/tensors/terms/zeros/zeros.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'ZEROS'
+Description: 'Initializing a new tensor with zeros.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+  - 'AI'
+Tags:
+  - 'AI'
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/machine-learning'
+---
+
+
+The `.zeros()` method returns a [Tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) of a specified shape with zeros. Optionally, the output tensor, datatype, layout, device, and whether autograd records operations can be specified.
```

Comment:
```suggestion
The **`.zeros()`** method returns a tensor of a specified shape filled with zeros. Optionally, the output tensor, its datatype, layout, device, and whether autograd records operations can be selected.
```

Reply:
Modifications: 
1. the first instance of a term should be in bold and backticks
2. There is no entry of tensors as of now, so removed the link